src/resources/lang/nl/crud.php

```diff
@@ -56,7 +56,7 @@
         'delete_confirmation_title'                   => 'Item verwijderd',
```

Comment:
Where did these all come from? (if these are a new lang file, shouldn't they be a separate PR?)